src/styles/card.scss

```
@@ -0,0 +1,89 @@
+.card {
+  width: 200px;
+  font-family: $base-font-family;
+  color: $primary-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

This param can be defined globally for all elements in main.scss
```suggestion
  * {
    box-sizing: border-box;
  }
```